BraintreeDataCollector/src/main/java/com/braintreepayments/api/DataCollector.java

```diff
@@ -136,6 +139,7 @@ public String getPayPalClientMetadataId(Context context) {
         return "";
     }
 
+    // this method can be removed, we won't be using it
```

Comment:
This method submits risk data to Magnes - do we no longer need to be doing that?